specification/dimensions/tags.md

```diff
@@ -0,0 +1,76 @@
+# Tags
+
+The Tags column represents the set of tags assigned to Tag Sources that also account for potential provider-defined or user-defined tag evaluations.  Tags are commonly used for scenarios like adding business context to billing data to identify and accurately allocate charges.
+
+A tag becomes *finalized* when a single value is selected from a set of possible tag values assigned to the tag key.  This can occur when a tag is set at different levels of a resource grouping hierarchy (if supported by the Provider), or a tag is value is determined from a set of user-defined rules.
+
+The Tags column adheres to the following requirements:
+
+* The Tags column MUST contain user-defined and/or provider-defined tags.
+* The Tags column MUST only contain finalized tags.
+* The Tags column MUST be in Key-Value Format.
+* A Tag without a specified value MUST have its value set to null.
```

Comment:
If clearer, lets go with this. Otherwise, ignore.
```suggestion
* A Tag key without a specified value MUST have its tag value set to null.
* If Tag finalization is supported, providers MUST publish tag finalization methods and semantics within their respective documentation.
```